sdk/nginx/arm-nginx/review/arm-nginx.api.md

```diff
@@ -129,13 +175,13 @@ export type ConfigurationsAnalysisResponse = AnalysisResult;
 
 // @public
 export interface ConfigurationsCreateOrUpdateOptionalParams extends coreClient.OperationOptions {
-    body?: NginxConfiguration;
+    body?: NginxConfigurationRequest;
     resumeFrom?: string;
     updateIntervalInMs?: number;
 }
 
 // @public
-export type ConfigurationsCreateOrUpdateResponse = NginxConfiguration;
+export type ConfigurationsCreateOrUpdateResponse = NginxConfigurationResponse;
```

Comment:
is this introduced by swagger change?

Reply:
yes, it's intorduced from https://github.com/Azure/azure-rest-api-specs/pull/30438/checks?check_run_id=31849540186